app/controllers/posts_controller.rb

```diff
@@ -48,8 +46,37 @@ def destroy
     end
   end
 
+  def api_index
+    @user = User.find(params[:user_id])
+    @posts = Post.where(author_id: @user.id).includes(:comments, :user).paginate(page: params[:page], per_page: 10)
+    render json: @posts
+  end
+
+  # API endpoint to list all comments for a user's post
+  def api_comments
```

Comment:
- You have done great so far :+1: But there are few issues you need to work on before moving to the next milestone. :+1: 

- [ ] Just as the previous reviewer pointed out, I still get errors of access denied when I try to access the API endpoints. I'll suggest you fix these and actually test them out using Postman before requesting for another review. :+1: :+1: 

- - Screenshot reference:
![Screenshot from 2023-12-23 23-58-51](https://github.com/maths29/blog_app/assets/107632780/4f6983a1-58a8-4c22-917c-4b24cec9bf82)


- [ ] Also this endpoint I pointed out in the screenshot below, is meant to return all the posts of a given user, therefore I don't think it needs a `post id`, its an index method, and the only needed parameter is the `users' id` and not a post id because you are requesting for all the posts and not an individual post like this `/api/users/:user_id/posts` :+1: :+1: :100: :wave: 

![Selection_079](https://github.com/maths29/blog_app/assets/107632780/d63af457-c1d7-4ebc-acd5-57f478f47937)